typescript-selenium-webdriver/README.md

```diff
@@ -2,15 +2,24 @@
 
 This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser, including how to suppress pre-existing or third-party failures using [Jest Snapshot Testing](https://jestjs.io/docs/en/snapshot-testing).
 
+## Getting Started
+
+The individual files in the sample contain comments that explain the important parts of each file in context.
+
+Some good places to start reading are:
+
+* [tests/index.test.ts](./tests/index.test.ts): The example tests in here demonstrate opening [src/index.html](./src/index.html) in a browser with Selenium and run accessibility scans with axe-webdriverjs

```

Comment:
It feels like the verbs should be (how to 'open' and 'run') or ('opening' and 'running')

Reply:
Used "that opens"/"that runs" for consistency with other bullets